static/js/stream_color.js

```diff
@@ -59,7 +59,7 @@ exports.set_colorpicker_color = function (colorpicker, color) {
 exports.update_stream_color = function (sub, color, opts) {
     opts = _.defaults({}, opts, {update_historical: false});
     sub.color = color;
-    const id = parseInt(sub.stream_id, 10);
+    const id = sub.stream_id;
```

Comment:
We should probably additionally change these `const id` variable names to `const stream_id`.

Reply:
I'm adding a commit for this on top, e2681372f3622dc8baa74cbbb3df7da5768f3f49